aten/src/ATen/core/Tensor.cpp

```diff
@@ -56,11 +56,12 @@ Tensor Tensor::tensor_data() const {
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 bool Tensor::is_view() const {
-  return impl::GetVariableHooks()->is_view(*this);
+  return impl_->view_meta() ? impl_->view_meta()->is_view() : false;
```

Comment:
Reviewer finger exercises 😁 : this can just be `impl_->view_meta() && impl_->view_meta()->is_view()`